code/src/index.css

```diff
@@ -1,5 +1,7 @@
 body {
```

Comment:
This file can be removed and the body styling can be moved to App.js or use styled-components helper such as[ { createGlobalStyle } ](https://styled-components.com/docs/api#createglobalstyle)